src/pydata_sphinx_theme/assets/styles/variables/_fonts.scss

```diff
@@ -41,3 +41,5 @@ html {
   --pst-font-family-heading: var(--pst-font-family-base-system);
   --pst-font-family-monospace: var(--pst-font-family-monospace-system);
 }
+
+$line-height-body: 1.65;
```

Comment:
I use this variable along with $navbar-link-padding-y to set the height in `.pst-navbar-icon`, which allows me to remove the wrapper spans around all the icons (i -> svg)